apps/prairielearn/elements/pl-code/pl-code.py

```diff
@@ -12,7 +12,7 @@
 import pygments.util
 from code_utils import parse_highlight_lines
 from pygments.styles import STYLE_MAP, get_style_by_name
-from pygments.token import Token
+from pygments.token import Token, _TokenType
```

Comment:
Only concern is whether this is type is part of the public interface. I'm not quite sure how the annotation would read otherwise though, so this is probably fine.

Reply:
We could always replace this with `Any` in the return type annotation? I guess that's a good fallback if this ever gets removed from the API.

Reply:
Yeah, it's pretty unlikely to get removed anyway. Imo we should just stick with the current annotation then switch if something weird happens.